pkg/test/harness.go

```diff
@@ -96,21 +98,28 @@ func (h *Harness) Config() (*rest.Config, error) {
 func (h *Harness) RunKUDO() error {
 	config, err := h.Config()
 	if err != nil {
+		log.Printf("Error getting harness config: %v\n", err)
 		return err
 	}
 
-	mgr, err := manager.New(config, manager.Options{
-		Scheme: testutils.Scheme(),
-	})
+	mgr, err := manager.New(config, manager.Options{})
 	if err != nil {
+		log.Printf("Error creating manager: %v\n", err)
+		return err
+	}
+
+	if err := apis.AddToScheme(mgr.GetScheme()); err != nil {
```

Comment:
I'm trying to debug an issue here, I need to finish this up and maybe make a separate PR for it